### ISSUE TITLE
css rules from svg style selector ignored

### ISSUE THREAD

**Issue by sverweij** [gabelerner/canvg#321]
**steps to reproduce**
Use this svg in the demo (https://gabelerner.github.io/canvg/examples.index.htm)
```svg
<svg width="480" height="60" xmlns="http://www.w3.org/2000/svg">
  <defs>
  <style type="text/css">
  svg {
    font-family: serif;
    font-size: 48pt;
    fill: blue;
  }
  </style>
</defs>
  <text y="50" x="6">serif 48pt</text>
</svg>
```
**expected**
Text is rendered in blue as 48pt serif:
![expected](https://cloud.githubusercontent.com/assets/4822597/5516214/d4b88d0e-8890-11e4-9d97-7581d824cda8.png)

**found**
Text is rendered in something else. It looks as if the rules between ```svg{ }``` were not applied: 
![found](https://cloud.githubusercontent.com/assets/4822597/5516215/e126486a-8890-11e4-8aaf-61a87dddc2dd.png)

**notes**
- In an older version (I have a 2013-08-24 timestamp on the file) canvg did use the "global" css style rules as described under 'expected'
- tested in firefox 34.0 and safari 8.0.2 (both on MacOSX 10.10.1)
- workarounds 
  - copy the style rules in the svg selector to selectors for individual elements (```text {yaddayadda}``` _is_ applied)
  - use attributes in the svg element instead of css style rules
  - (keep using the older version for a while)
- screen shot from demo [in attachment](https://cloud.githubusercontent.com/assets/4822597/5516216/04e7fec4-8891-11e4-9a6f-6d2644a48209.png)

**Comment by gabelerner** [comment]
closed by https://github.com/gabelerner/canvg/pull/341

**Comment by mato75** [comment]
this is not fixed.

**Comment by thelordgabo** [comment]
yes this is not fixed.

**Comment by thelordgabo** [comment]
Will be good to give support for css rules also not only for inline styles.

**Comment by sverweij** [comment]
Still looks fixed to me...

Alarmed this apparently broke since it was fixed in February, I checked the original reproduction sample  against the current version (http://gabelerner.github.io/canvg/examples/ ). Result is 100% OK, though:
![screen shot 2016-01-06 at 17 58 14](https://cloud.githubusercontent.com/assets/4822597/12148523/3881dd00-b49f-11e5-8aad-83a2e65ff2cf.png)

That said you probably _did_ run into unexpected behavior with an svg that looked similar. Try and run it against the current version (link above) and if the problem persists, log an issue with the offending svg.
